internal/bucket/azureBucket_integration_test.go

```
@@ -0,0 +1,198 @@
+func TestAzuriteSASUploadRequiresBlockBlobHeader(t *testing2.T) {
+	b := setupAzuriteBucket(t)
+	uploadURL, err := b.RequestUploadUrlForFileUpdate("app-1", "production", "1", "1674170951", "bundles/android.js")
+	require.NoError(t, err)
+
+	// Without the header Azure refuses the PUT: the exact failure eoas hit
+	// before the server started returning upload headers.
+	reqWithoutHeader, err := http.NewRequest(http.MethodPut, uploadURL, strings.NewReader("payload"))
+	require.NoError(t, err)
+	respWithoutHeader, err := http.DefaultClient.Do(reqWithoutHeader)
+	require.NoError(t, err)
+	respWithoutHeader.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `respWithoutHeader.Body.Close` is not checked

(errcheck)